fuzzers/qemu_systemmode/build.rs

```diff
@@ -5,7 +5,7 @@ macro_rules! assert_unique_feature {
     () => {};
     ($first:tt $(,$rest:tt)*) => {
         $(
-            #[cfg(all(not(any(doc, feature = "clippy")), feature = $first, feature = $rest))]
+            #[cfg(all(not(any(doc, clippy)), feature = $first, feature = $rest))]
```

Comment:
@rmalmain As I changed it now it is not what you wanted to achieve?

`#[cfg(clippy)]`: https://blog.rust-lang.org/2024/02/28/Clippy-deprecating-feature-cargo-clippy.html

Reply:
no no it's all good, you can keep it like this.

Reply:
@tokatoka this doesn't work for you?

Reply:
i tried changing all occurence of them but they don't work
cargo clippy --all-features in libafl-qemu folder fails